worker/src/polkadex_db/balances.rs

```diff
@@ -83,6 +84,32 @@ impl<D: PermanentStorageHandler> BalancesMirror<D> {
     pub fn take_disk_snapshot(&mut self) -> Result<Vec<u8>> {
         self.general_db.write_disk_from_memory()
     }
+
+    pub fn load_disk_snapshot(&mut self) -> Result<()> {
+        if self.general_db.read_disk_into_memory().is_err() {
+            return Err(PolkadexDBError::_KeyNotFound);
+        }
+        Ok(())
+    }
+
+    pub fn prepare_for_sending(&self) -> Result<Vec<BalancesData>> {
```

Comment:
Can you write unit tests for this function? Will probably be similiar like the ones for `load_disk_snapshot`

Reply:
Tests written for all mirrors.